src/openrct2/scripting/Plugin.cpp

```diff
@@ -161,4 +162,57 @@ PluginType Plugin::ParsePluginType(const std::string_view& type)
     throw std::invalid_argument("Unknown plugin type.");
 }
 
+static std::vector<std::string> _validLicenses{ "0BSD",
```

Comment:
Like I said earlier, I don't think we should force any licence upon the plug-in creator. The reason to create this field is to force plug-ins to make a decision in the first place.

I say change this to only check if the field is set.

Reply:
I agree. We shouldn't limit this field to particular licences only. If someone wants to use a novelty licence (the "buy me a beer licence" comes to mind), they should be able to do so.

Reply:
Ok, I'll drop it later to let more people comment on it first.